pkg/R/utils.R

```diff
@@ -164,11 +176,11 @@ reserialize <- function(rdd) {
   if (!inherits(rdd, "RDD")) {
```

Comment:
Let's update the comment of this function to explicitly say it reserializes to Byte mode.